File: library/scene_navigation/src/test/java/com/bytedance/scene/navigation/utility/TestActivity.java
package com.bytedance.scene.navigation.utility;

import android.app.Activity;
import android.os.Bundle;
import android.widget.FrameLayout;

import androidx.annotation.Nullable;

public class TestActivity extends Activity {
    public FrameLayout mFrameLayout;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        mFrameLayout = new FrameLayout(this);
        setContentView(mFrameLayout);
    }
}

File: library/scene_ktx/src/test/java/com/bytedance/scene/ktx/utility/TestActivity.kt
package com.bytedance.scene.ktx.utility

import android.app.Activity
import android.os.Bundle
import android.widget.FrameLayout

class TestActivity : Activity() {
    lateinit var mFrameLayout: FrameLayout

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        mFrameLayout = FrameLayout(this)
        setContentView(mFrameLayout)
    }
}